training/distributed_training/pytorch/data_parallel/roberta/roberta.ipynb

```diff
@@ -0,0 +1,418 @@
+{
```

Comment:
<p>What's the model you use between the two (1.3B and 350M) by default in this notebook? Can we make if statement to pick up the right hyperparameters depending on an input string, something like </p><pre class="ql-syntax" spellcheck="false">if model_name=='roberta-l-1_3billion':
  hyperparameters= # the first hyperparameter dict
else:
  hyperparameters= # the second hyperparameter dict
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3420/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>